governance/votes/Executive Vote - March 11, 2022.md

```diff
@@ -0,0 +1,116 @@
+* [Price Change Interval (`step`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-auction-price-function#exponential-stair-step): 90 seconds
+* [Auction Price Multiplier (`buf`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-auction-price-multiplier): 1.30
+* [Liquidation Penalty (`chop`)](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-penalty): 13%
+
+-- Limits --
+
+* [Local Liquidation Limit (`ilk.hole`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-local-liquidation-limit): 3 million DAI
+* [Maximum Auction Drawdown (`cusp`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-max-auction-drawdown): 0.4
+* [Maximum Auction Duration (`tail`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-max-auction-duration): 140 minutes
+* [Breaker Price Tolerance (`tolerance`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-breaker-price-tolerance): 0.5
+
+-- Incentives --
+
+* [Proportional Kick Incentive (`kick`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-proportional-kick-incentive): 0.1%
+* [Flat Kick Incenctive (`tip`)](https://manual.makerdao.com/parameter-index/collateral-auction/param-flat-kick-incentive): 300 DAI
+
+Please review the following forum threads containing information about CRVV1ETHSTETH-A to inform your position before voting.
+
+* [Proposal Thread](https://forum.makerdao.com/t/curvelp-steth-eth-mip6-collateral-onboarding-application/10229)
+* [Risk Core Unit Evaulation](https://forum.makerdao.com/t/curvelp-steth-eth-collateral-onboarding-risk-evaluation/11224)
+* [Protocol Engineering Coure Unit Evaluation](https://forum.makerdao.com/t/curve-fi-eth-steth-erc20-token-smart-contract-technical-assessment/11855)
+* [Oracles Core Unit Evaluation](https://forum.makerdao.com/t/curvelp-steth-eth-collateral-onboarding-oracle-assessment-mip10c3-sp43/11854)
+
+### Rate 
+
+As per these successful governance polls [1](https://vote.makerdao.com/polling/QmagLVA2), [2](https://vote.makerdao.com/polling/Qmdd4Pg7) a new rate limiter, detailed below, will be added to the flapper contract if this executive proposal passes.
+
+* [Modify the Flapper-Contract](https://forum.makerdao.com/t/rate-limited-flapper/13056) by introducing a rate limit so the protocol can control the amount of parallel running flap-auctions.
+* Add a parameter, `lid`, which sets the total value of DAI allowed to be sent out to [surplus (flap) auctions](https://auctions.makerdao.com/flap) at once.
+* Set the `lid` parameter to **150,000 DAI**.
+* Re-activate [surplus (flap) auctions](https://auctions.makerdao.com/flap).
+
+Please note that there will be no changes to the [Surplus Auction Buffer](https://manual.makerdao.com/parameter-index/core/param-system-surplus-buffer) as as result of this executive proposal.
```

Comment:
No changes 'at this time'?

Reply:
Maybe it's covered by results thing.

Reply:
System Surplus Buffer rather than Surplus Auction Buffer?

Reply:
Fixed